sensing/preprocessor/gnss/gnss_poser/src/gnss_poser_core.cpp

```diff
@@ -109,18 +91,20 @@ void GNSSPoser::callbackNavSatFix(const sensor_msgs::NavSatFix::ConstPtr & nav_s
   }
 
   // generate gnss_antenna_pose_msg
-  geometry_msgs::PoseStamped gnss_antenna_pose_msg;
+  geometry_msgs::msg::PoseStamped gnss_antenna_pose_msg;
   gnss_antenna_pose_msg.header.stamp = nav_sat_fix_msg_ptr->header.stamp;
   gnss_antenna_pose_msg.header.frame_id = map_frame_;
   gnss_antenna_pose_msg.pose.position = median_position;
   gnss_antenna_pose_msg.pose.orientation = orientation;
 
   // get TF from base_link to gnss_antenna
-  geometry_msgs::TransformStamped::Ptr TF_base_to_gnss_ptr(new geometry_msgs::TransformStamped);
-  getStaticTransform(gnss_frame_, base_frame_, TF_base_to_gnss_ptr, nav_sat_fix_msg_ptr->header.stamp);
+  geometry_msgs::msg::TransformStamped::SharedPtr TF_base_to_gnss_ptr(
```

Comment:
Nitpick: If you are using the Google Style guide, variable names should not be uppercase. Also, make_shared can be used here.

Reply:
good catch. Since we use c++14, we should use `make_shared` for exception safety and to save potentially one memory allocation Good post [here](https://arne-mertz.de/2018/09/make_shared-vs-the-normal-shared_ptr-constructor/)